src/lib/mina/transaction.ts

```diff
@@ -114,6 +115,25 @@ type Transaction<
    * ```
    */
   safeSend(): Promise<PendingTransaction | RejectedTransaction>;
+
+  /**
+   * Modifies a transaction to set the fee to the new fee provided. Because this change invalidates proofs and signatures both are removed. The nonce is not increased so sending both transitions will not risk both being accepted.
+   * @returns {TransactionPromise<false,false>} The same transaction with the new fee and the proofs and signatures removed.
+   * @example
+   * ```ts
+   * tx.send();
+   * // Waits for some time and decide to resend with a higher fee
+   *
+   * tx.setFee(newFee);
+   * await tx.sign([feePayerKey]));
+   * await tx.send();
+   * ```
+   */
+  setFee(newFee:UInt64) : TransactionPromise<Proven,false>;
+  /**
+   * setFeePerSnarkCost behaves identically to {@link setFee} but the fee is given per Account Update in the transaction. This is useful because nodes prioritize transactions by fee per weight unit.
```

Comment:
> but the fee is given per Account Update in the transaction. This is useful because nodes prioritize transactions by fee per weight unit.

This seems to describe the previous behavior.  Should be updated to mention snark cost instead of weight unit, and mention proofs as well as AccountUpdates.  Might be nice to `{@link getTotalTimeRequired}` as well, but I'm not sure that's a public function.

Reply:
The link at least works in my LSP